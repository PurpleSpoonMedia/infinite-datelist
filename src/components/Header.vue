<template>
  <header class="header text-center py-4 mb-4">
    <router-link to="/"><h2>VueJS CRUD with PHP/MySQL</h2></router-link>
  </header>
</template>

<style lang="scss" scoped>
.header {
  border-bottom: 1px solid #ccc;
}
</style>

