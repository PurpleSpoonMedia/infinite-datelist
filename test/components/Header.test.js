import { shallowMount, RouterLinkStub } from '@vue/test-utils';
import Header from '../../src/components/Header.vue';

describe('Header.vue', () => {
  test('should render Header correctly', () => {
    const wrapper = shallowMount(Header, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  })
})
