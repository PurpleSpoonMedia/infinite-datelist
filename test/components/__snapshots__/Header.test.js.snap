// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue should render Header correctly 1`] = `
<header class="header text-center py-4 mb-4">
  <a>
    <h2>VueJS CRUD with PHP/MySQL</h2>
  </a>
</header>
`;
